module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -907,7 +897,7 @@ public function setCatUsername(?string $catUsername): UserEntityInterface
      */
     public function getCatUsername(): ?string
     {
-        return $this->cat_username;
+        return $this->cat_username ?? '';
```

Comment:
If we don't have a default value here, Laminas\Db will throw an exception if you call the getter on a row that hasn't been persisted yet, which was causing some test failures after refactoring.